npm/private/npm_translate_lock.bzl

```diff
@@ -598,9 +597,11 @@ load("@aspect_rules_js//npm/private:npm_linked_packages.bzl", "npm_linked_packag
                     }
 
     if fp_links:
-        defs_bzl_header.append("""load("@aspect_rules_js//npm/private:npm_link_package.bzl",
+        defs_bzl_header.append("""load(
+    "@aspect_rules_js//npm/private:npm_link_package.bzl",
```

Comment:
nit: unrelated refactoring?

Reply:
to satisfy lint; not sure how it got out of sync; but I'll split out the example into its own commit with proper attribution and then this change will go with that.